moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Box, Grid, Typography } from "@mui/material";
+
+/**
+ * ProjectSummaryCouncilDistricts Component
+ * @param {Object} data - The project geography from the graphql query's data object
+ * @param {Object} classes - The shared style settings
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const ProjectSummaryCouncilDistricts = ({ projectGeography, classes }) => {
+  // reduce the array of geography objects into an array of city council districts
+  const getDistricts = (data) => {
+    const initialValue = [];
+    const districts = data.reduce(
+      (acc, component) => [...acc, component["council_districts"]],
+      initialValue
+    );
+
+    return [...new Set(districts.flat())];
+  };
+
+  return (
+    <Grid item xs={12} className={classes.fieldGridItem}>
```

Comment:
Small thing but this Grid item wrapper probably could be removed since this component is wrapped with one where it is rendered in `ProjectSummary`. Might help make it more portable if we ever show this elsewhere but not super important. 🤷

Reply:
i noticed the same thing, but it looks like most of our summary components are nested Grids, so I was worried it was going to not behave like the other ones

Reply:
cool - that makes more sense! Didn't check the other ones